core/crypto/src/signature.rs

```diff
@@ -940,18 +936,18 @@ mod tests {
 
         let sk = SecretKey::from_seed(KeyType::SECP256K1, "test");
         let pk = sk.public_key();
-        let expected = "\"secp256k1:BtJtBjukUQbcipnS78adSwUKE38sdHnk7pTNZH7miGXfodzUunaAcvY43y37nm7AKbcTQycvdgUzFNWsd7dgPZZ\"";
+        let expected = "\"secp256k1:2xVqteU8PWhadHTv99TGh3bSffGb32cFokopSg8xPAiFVjv5UvRUYfNPzmZctBuXryhQosc9dfY1TKKrV4YiyiZH\"";
```

Comment:
This looks a bit sus.  Do we depend anywhere in the protocol on how `from_seed` generates keys?

Reply:
@mm-near, do you know of any issues we might face if we switch the RNGs in secp256k1 package? From what I’m seeing this is only affecting tests and the PR LGTM.